docs/guides/src/main/server/logging.adoc

```diff
@@ -7,102 +7,116 @@ title="Configuring logging"
 summary="Learn how to configure Logging"
 includedOptions="log-*">
 
-Keycloak uses the jboss logmanager logging framework. The high-level overview for the available log handlers is shown below:
+Keycloak uses the jboss logmanager logging framework. The following is a high-level overview for the available log handlers:
 
 * root
 ** console (_default_)
 ** file
 
 == Logging: Root configuration
```

Comment:
@DGuhr I am wondering if "Root" should be removed  because this section also includes category-specific logging configuration.

Reply:
Makes sense. sry 4 late response here, too.